### PR TITLE
Add Nabarun and Arnaud to GitHub Admin Team

### PR DESCRIPTION
**Pull request by nikhita**
Ref: [sig contribex thread](https://groups.google.com/g/kubernetes-sig-contribex/c/O26DCoCdMYQ/m/GwkkzX-2AQAJ) for the nomination.

Lazy consensus achieved.

/hold

/committee steering
/cc @kubernetes/steering-committee 
need majority of steering committee to lgtm this change

/assign @cblecker @mrbobbytables 
currently active gh admins (Ihor is temporarily unavailable)
Can you also +1 here?

/assign @palnabarun @ameukam 
Can you confirm that you are interested in joining the GitHub Admin Team and accept the security embargo policy?

### CODE REVIEW

File: github-management/README.md
Comment (MadhavJivrajani) [inline github-management/README.md:30]: Small typo
```suggestion
* Arnaud Meukam (**[@ameukam](https://github.com/ameukam)**, Central European Time)
```

**Review by justaugustus — APPROVED** [review]
+1 (Steering hat) 🎉 
nit: could we alpha-sort reviewers/approvers?

**Comment by palnabarun** [comment]
> Can you confirm that you are interested in joining the GitHub Admin Team and accept the security embargo policy?

I confirm that I am interested to join the GitHub Admin Team and I solemnly swear to abide by the guidelines set forth in the Security Release Process.

**Comment by dims** [comment]
LGTM!!

**Comment by liggitt** [comment]
+1

**Comment by nikhita** [comment]
>  nit: could we alpha-sort reviewers/approvers?

done

**Comment by nikhita** [comment]
@mrbobbytables ping for a +1

we'll have steering majority + gh admin sign off after you lgtm.

**Comment by mrbobbytables** [comment]
+1 with both hats on :)

**Comment by nikhita** [comment]
Welcome, @palnabarun and @ameukam!! 🚀 

/hold cancel

**Comment by nikhita** [comment]
Ah, this still needs a lgtm label. We have all the required +1s.

Looking for a lgtm now.

**Pull request merged**
